packages/documentation/pages/usage/components/table.vue

```diff
@@ -906,14 +906,14 @@ _Notes_:
 		</KtTableConsumer>
 		<KtTableConsumer #default="{ columns, orderBeforeColumn }">
 			<div>
-				<KtInput
+				<input
 					label="Drag From: "
 					type="number"
 					min="0"
 					:max="columns.length-1"
 					v-model="fromIndex"
 				/>
-				<KtInput
+				<input
```

Comment:
I have another PR that replaces this with KtFieldNumber in the making but the KtFieldNumber errors on mounting because of values that don't match min and max and this was the easiest replacement for the purpose of KtInput removal

Reply:
LGTM